packages/query-core/src/subscribable.ts

```diff
@@ -16,7 +16,7 @@ export class Subscribable<TListener extends Function> {
     }
   }
 
-  hasListeners(): boolean {
+  protected hasListeners(): boolean {
```

Comment:
There is no use case using hasListeners as public method.

Reply:
we're about to use it in a separate PR 😅